source/common/config/filter_json.cc

```diff
@@ -378,21 +378,9 @@ void FilterJson::translateTcpProxy(
   json_config.validateSchema(Json::Schema::TCP_PROXY_NETWORK_FILTER_SCHEMA);
 
   JSON_UTIL_SET_STRING(json_config, proto_config, stat_prefix);
+  JSON_UTIL_SET_STRING(json_config, proto_config, cluster);
```

Comment:
I don't think we need to add this. v1 config is deprecated and we're not adding new fields to it.